### ISSUE TITLE
NTR: Gastric biopsy: intraepithelial lymphocytosis (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8661]
New term request Gastric biopsy: intraepithelial lymphocytosis (nadea3_200213183508): Lymphocytic  UMLS: 2697799

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000250
- label: Gastric intraepithelial lymphocytosis
- parents: [Abnormal gastric mucosa morphology (HP:0004295)](https://hpo.jax.org/app/browse/term/HP:0004295)
- pmid: PMID:8669535
- def: The count of lymphocytes located in the epithelial layer of the gastric mucosa is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8661
- synonyms:
